static/app/views/performance/transactionSummary/transactionOverview/tagExplorer.tsx

```diff
@@ -303,10 +304,7 @@ class _TagExplorer extends React.Component<Props> {
         organization_id: parseInt(organization.id, 10),
       });
 
-      const searchConditions = new MutableSearch(eventView.query);
-
-      // remove any event.type queries since it is implied to apply to only transactions
-      searchConditions.removeFilter('event.type');
+      const searchConditions = normalizeSearchConditions(eventView.query);
```

Comment:
same here, `transaction` filter is also removed. not sure if it's intentional

Reply:
@udameli Yes, it's intentional.

We do not want to sneak in `transaction:<name>` search conditions since it's already implied. This should only affect search queries performed within the transaction summary detail page.

Reply:
sounds good! thanks for the context